databaseFiles/helperMethods.js

```diff
@@ -0,0 +1,5 @@
+//const Sequelize = require('sequelize');
+
+/**
+ * Method: addEntry(tableName,)
+ */
```

Comment:
It was destined to be a file for methods to make things a bit more straightforward to use. I'll go ahead and remove it from the commit because as much as I still want to do it I know I won't get around to it